sn_cli/benches/files.rs

```diff
@@ -79,7 +82,10 @@ fn criterion_benchmark(c: &mut Criterion) {
 
         let mut group = c.benchmark_group(format!("Upload Benchmark {}MB", size));
         group.sampling_mode(criterion::SamplingMode::Flat);
-        group.measurement_time(Duration::from_secs(120));
+        // One sample may compose of multiple iterations, and this is decided by `measurement_time`.
+        // Set this to a lower value to ensure each sample only contains one iteration.
+        // To ensure the download throughput calculation is correct.
+        group.measurement_time(Duration::from_secs(5));
```

Comment:
Then the sampling could be off due to an aberration... We want many runs for sounder statistical analysis. 

I'm not sure why this is needed? (same for change below)